src/components/Tabs.jsx

```diff
@@ -38,29 +52,56 @@ class Tabs extends React.Component {
         })
     }
 
+    _getTabForValue = (value) => {
+        const c = React.Children.toArray(this.props.children)
+        if (typeof value == 'number') return c[value] || c[0]
```

Comment:
I think this will break as soon as someone uses numeric values (e.g. ids) as `value` prop. I think there are two possible solutions:
1. We mark the `value` prop as required
2. We are cloning every child that doesn't have a `value` prop and set it to it's index

Advantage of *2* is that it's a non-breaking change but I'm a bit worried about possible performance implications if we cloning children on every render. 
What's your opinion? Or do you have a better idea?

Reply:
@henningmu 
What do you think about the solution: If `onChange` or `defaultTab` prop is set, each child is required to have `value` set. 

This way, we either get children by index or by matching value, but not both at the same time.

Reply:
Sounds like a plan 👍
Should we log / error to console in case this condition isn't satisfied? Seems to be too complex to be modeled with propTypes 😋

Reply:
@henningmu I tend to prefer throwing an exception so that developer could spot this error during development time. What do you think